src/index.js

```diff
@@ -58,15 +69,19 @@ function eachMethod(obj, callback) {
   }
 }
 
-export function combo(...shortcuts) {
+export function combo(shortcuts, runInsideInputs) {
```

Comment:
This method signature change is a breaking change. Suggest to keep old signature, then check last param.

```js
combo(...shortcuts) {
  let runInsideInputs = false;
  if (shortcuts.length && typeof shortcuts[shortcuts.length - 1] === 'boolean') {
    runInsideInputs = shortcuts.pop();
  }
  //...
}
```

That will enable
```js
@combo('ctrl+f', 'command+f', true)
```